include/SFML/Graphics/CircleShape.hpp

```diff
@@ -105,6 +105,18 @@ class SFML_GRAPHICS_API CircleShape : public Shape
     ////////////////////////////////////////////////////////////
     Vector2f getPoint(std::size_t index) const override;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the geometric center of the circle
+    ///
+    /// The returned point is in local coordinates, that is,
+    /// the shape's transforms (position, rotation, scale) are
+    /// not taken into account.
+    ///
+    /// \return The geometric center of the shape
+    ///
+    ////////////////////////////////////////////////////////////
```

Comment:
If we don't change the doxygen comment, then we might as well just copy the docs from the parent class with:

```
    ////////////////////////////////////////////////////////////
    /// \copydoc sf::Shape::getGeometricCenter()
    ///
    ////////////////////////////////////////////////////////////
```